### PR TITLE
Fix image placement in `Testing`

### PR DESCRIPTION
**Pull request by itscynxx**
Fixes a misplaced image in the page for manually downloading development Northstar code

### CODE REVIEW

File: docs/development/testing.md [hunk 1]
```diff
@@ -46,14 +46,15 @@ The tool is still being improved upon but already more than usable. Check its RE
 
 Before starting, make sure you have a **working and up-to-date Northstar install** and you're **logged into GitHub with your GitHub account** (downloading files from CI only works while logged into any GitHub account)!
 
-1. Head to the bottom of the page of the PR and click on "_Show all checks_" ![rcon1](https://user-images.githubusercontent.com/40122905/179726100-48945eb6-3ebe-467f-acef-1c7d56f3e4bd.png)
-2. For "_CI / build (pull\_request)_" click on "_Details_"\
+1. Head to the bottom of the page of the PR and click on "_Show all checks_"\
+   &#x20;![rcon1](https://user-images.githubusercontent.com/40122905/179726100-48945eb6-3ebe-467f-acef-1c7d56f3e4bd.png)
```
Comment (GeckoEidechse) [inline docs/development/testing.md:50]: What's up with that `&#x20;`? ^^"

Also seems to have accidentally updated the numbering, lol

Comment (itscynxx) [reply docs/development/testing.md:50]: > What's up with that `&#x20;`? ^^"
> 
> Also seems to have accidentally updated the numbering, lol

Honestly, don't know. The rest of the images just have them
Likely, it's to set an offset so the images are aligned with the start of the text, as the text is a number list thing

**Comment by GeckoEidechse** [comment]
If you don't mind I'll split this up a bit for my own sake ^^

**Review by GeckoEidechse — APPROVED** [review]
Changes from review addressed and some merged via separate commits
- 836c0ac92f3c42c5d358679926cb5bfd76921e70
- a52ed59491359fddc2ba61a0709641896e4c4df1

**Pull request merged**